pkg/controller/controller_test.go

```diff
@@ -1709,9 +1838,12 @@ func runProvisionTest(t *testing.T, k string, tc provisioningTestcase, requested
 	clientSet := fakeclientset.NewSimpleClientset(tc.clientSetObjects...)
 
 	pluginCaps, controllerCaps := provisionCapabilities()
-	csiProvisioner := NewCSIProvisioner(clientSet, 5*time.Second, "test-provisioner", "test", 5, csiConn.conn,
-		nil, provisionDriverName, pluginCaps, controllerCaps, supportsMigrationFromInTreePluginName, false, csitrans.New(), nil, nil, nil, nil, nil, tc.withExtraMetadata)
 
+	if tc.skipDefaultFSType {
+		defaultfsType = ""
```

Comment:
I think the unit test is failing because this is modifying a global variable.

Reply:
@msau42 Thanks! Looks like that.. after untangling the test case , its working as expected!